src/adapters/api-blueprint.es6

```diff
@@ -57,9 +57,29 @@ function resourceShorthand(resource) {
           resource.transitions.get(0).computedHref);
 }
 
+/*
+ * Attempt to print out a pretty version of some input. Currently supported
+ * inputs include:
+ *
+ * - JSON
+ *
+ */
+function pretty(input) {
+  let prettified;
+
+  try {
+    prettified = JSON.stringify(JSON.parse(input), null, 2);
+  } catch (err) {
+    prettified = input;
+  }
+
+  return prettified;
+}
+
 swig.setFilter('indent', indent);
 swig.setFilter('bodyOnly', bodyOnly);
 swig.setFilter('resourceShorthand', resourceShorthand);
+swig.setFilter('pretty', pretty);
```

Comment:
I left it open to support XML and other formats in the future if we want. I'd prefer a generic pretty filter that detects the input and tries to make it pretty instead of adding logic to the template to determine which type of input it is.